habitat-baselines/habitat_baselines/config/experiments_hab3/rearrange_fetch_human_planner.yaml

```diff
@@ -0,0 +1,144 @@
+  video_dir: ${hydra:sweep.dir}/${hydra:sweep.subdir}/video
+  tensorboard_dir: ${hydra:sweep.dir}/${hydra:sweep.subdir}/tb
+  eval_ckpt_path_dir: ${hydra:sweep.dir}/${hydra:sweep.subdir}/checkpoints
+  checkpoint_folder: ${hydra:sweep.dir}/${hydra:sweep.subdir}/checkpoints
+  log_file: ${hydra:sweep.dir}/${hydra:sweep.subdir}/train.log
+  evaluate: False
+  writer_type: 'tb'
+  wb:
+    project_name: 'hab3'
+    entity: 'andrew-colab'
+    run_name: ${hydra:job.name}_${now:%Y-%m-%d}_${now:%H-%M-%S}
+
+
+  num_environments: 1
+  eval:
+    should_load_ckpt: True
+    video_option: ["disk"]
+  rl:
+    ddppo:
+      reset_critic: False
+    policy:
+      agent_0:
+        hierarchical_policy:
+          high_level_policy:
+            name: "FixedHighLevelPolicy"
+          # Override to use the oracle navigation skill (which will actually execute navigation).
+          defined_skills:
+            nav_to_obj:
+              skill_name: "OracleNavPolicy"
+              obs_skill_inputs: ["obj_start_sensor", "abs_obj_start_sensor", "obj_goal_sensor", "abs_obj_goal_sensor"]
+              max_skill_steps: 300
+              ignore_grip: True
+      agent_1:
+        hierarchical_policy:
+          high_level_policy:
+            name: "FixedHighLevelPolicy"
+            add_arm_rest: False
+          # Override to use the oracle navigation skill (which will actually execute navigation).
+          defined_skills:
+            nav_to_obj:
+              skill_name: "OracleNavPolicy"
+              obs_skill_inputs: ["obj_start_sensor", "abs_obj_start_sensor", "obj_goal_sensor", "abs_obj_goal_sensor"]
+              max_skill_steps: 300
+              ignore_grip: True
+
+habitat:
+  gym:
+    obs_keys:
+      - agent_0_head_depth
+      - agent_0_relative_resting_position
+      - agent_0_obj_start_sensor
+      - agent_0_obj_goal_sensor
+      - agent_0_obj_start_gps_compass
+      - agent_0_obj_goal_gps_compass
+      - agent_0_joint
+      - agent_0_is_holding
+      - agent_0_ee_pos
+      - agent_0_localization_sensor
+      - agent_1_head_depth
+      - agent_1_relative_resting_position
+      - agent_1_obj_start_sensor
+      - agent_1_obj_goal_sensor
+      - agent_1_obj_start_gps_compass
+      - agent_1_obj_goal_gps_compass
+      - agent_1_is_holding
+      - agent_1_ee_pos
+      - agent_1_localization_sensor
+
+  task:
+    task_spec: multi_agent_tidy_house_solved
+    actions:
+      agent_0_arm_action:
+        grip_controller: MagicGraspAction
+      agent_1_oracle_nav_action:
+        motion_control: human_joints
+        spawn_max_dist_to_obj: -1
+    measurements:
+      gfx_replay_measure:
+        type: GfxReplayMeasure
+      composite_success:
+        must_call_stop: False
+
+  environment:
+    max_episode_steps: 700
+
+  simulator:
+    kinematic_mode: True
+    ac_freq_ratio: 1
+    step_physics: False
+    agents:
+      agent_1:
+        articulated_agent_urdf: 'data/humanoids/humanoid_data/female2_0.urdf'
```

Comment:
The documentation mentions that we need to extract files in `data/` and some config files point there. Should this be `data/female2_0.urdf`?